README.hpcgap.md

```diff
@@ -0,0 +1,14 @@
+# HPC-GAP
+
+The beta release of GAP 4.9 includes experimental code to support multithreaded
+programming in GAP. GAP and HPC-GAP codebases diverged during the project, and
+we are currently working on unifying the codebases and incorporating the HPC-GAP
+code back into the mainstream GAP versions.
+
+This is work in progress, but including it into GAP 4.9 release will allow to
+conduct further code refactoring more efficiently, and will expose some new
+code to make visible new names and keywords. 
```

Comment:
This is our reason, but I am not sure it is helpful to tell end users. Perhaps better to say something more like this:
> This is work in progress, and HPC-GAP as it is included with GAP right now still suffers from various limitations and problems, which we are actively working on to resolve. However, including it with GAP (disabled by default) considerably simplifies development of HPC-GAP. It also means that you can very easily get a (rough!) sneak peak of HPC-GAP.

Well.. or something in this line... I don't claim what I wrote is great, sorry :/